packages/lib/SyncTargetJoplinCloud.ts

```diff
@@ -46,7 +51,18 @@ export default class SyncTargetJoplinCloud extends BaseSyncTarget {
 	}
 
 	public async isAuthenticated() {
-		return true;
+		try {
+			const fileApi = await this.fileApi();
+			const api = fileApi.driver().api();
+			const sessionId = await api.sessionId();
+			return Boolean(sessionId);
+		} catch (error) {
+			return false;
```

Comment:
If we silence an error we need to leave a comment as to why it's been done, as this should be the exception rather than the rule. And should it actually be silenced?

Reply:
I see what you mean. 
If we don't catch the error here we can't open the login window when the user presses "Synchronise" button. 

I change the code to this, see if it makes more sense now. Do you think it is necessary to add a comment here?
```ts
                ...
		} catch (error) {
			if (error.code === 403) return false;
			throw error;
		}
```

Reply:
That looks good like this